cluster/src/shard_table_manager.rs

```diff
@@ -43,6 +43,11 @@ impl ShardTableManager {
         self.inner.read().unwrap().contains_shard(shard_id)
```

Comment:
In fact, I think `ShardTableManager` is just a struct like xxx_map. How about name it like`ShardToTablesMap`? And rename `update_tables_of_shard` to `insert_or_update`, `remove_shard` to `remove` ?

Reply:
I don't think it is a good practice to suffix a struct with its type. Furthermore, current `ShardTableManager` provides more functionalities than the mapping from shard to tables, e.g. `find_table_by_name`. In conclusion, such renaming is not a good idea for me.